packages/modul-components/src/filters/postal-code/postal-code.ts

```diff
@@ -0,0 +1,34 @@
+import { PluginObject } from 'vue';
+import { MPostalCodeCountry } from '../../components/postalcodefield/postalcodefield';
+import { POSTAL_CODE_NAME } from '../filter-names';
+
+export class PostalCodeFilter {
+    static format(text: string, isoCountry: string): string {
```

Comment:
j'ajouterais de la doc ici, genre les cas supportés et les comportements avec des exemples..
/**
if postal code for : 
( html pour faire une belle liste)
CA : out put : "XXX XXX", does not check for number vs letters
US : less then 5 characters : one block
       >5 characters : XXXXX-XXXX* max 9 ?
...
*/

Reply:
Bon point! Je l'avais fait pour le filtre de téléphone que j'ai copié.
J'ai enlevé la doc de téléphone, sans la remplacer, comme un idiot...